g_src/enabler.h

```diff
@@ -33,8 +33,8 @@ using std::queue;
 #include <SDL2/SDL.h>
 #include <SDL2/SDL_thread.h>
 #ifdef __APPLE__
-# include <SDL_ttf/SDL_ttf.h>
-# include <SDL_image/SDL_image.h>
+# include <SDL2/SDL_ttf.h>
```

Comment:
While this will be correct once the SDL2 version's running properly, it's not correct now, though I guess it probably doesn't matter whether it's correct since you can't compile it either way (and yeah, it wouldn't take much to convince me to merge despite this issue, just think it needs addressing first)